share/scripts/sdcdb.tcl

```diff
@@ -156,7 +157,7 @@ proc sdcdb {args} {
     if {[catch {set result [dispatcher {*}$args]} msg]} {
         variable debug
         if {$debug} {
-            puts stderr $::errorInfo
+            debug_out stderr $::errorInfo
```

Comment:
What I meant:
you can leave out `variable debug` and `if ${debug} {`, that's the responsibility of `debug_out`.